app/src/main/java/com/edulectronics/tinycircuit/Views/CircuitActivity.java

```diff
@@ -190,7 +192,7 @@ public boolean onTouch (View v, MotionEvent ev) {
             Resources r = getResources();
             Component component = ((GridCell)((IDragSource) v)).getComponent();
             if(component != null) {
-                wireController.wire(component, ev, r.getInteger(R.integer.cell_size));
+                wireController.wire(component, ev);
```

Comment:
wire() staat niet in deze pull request maar lijkt me geen goede naam voor een methode